server/network.go

```diff
@@ -120,12 +134,13 @@ func newLocalNetwork(opts localNetworkOptions) (*localNetwork, error) {
 		nodeNames[i] = nodeName
 		cfg.NodeConfigs[i].Name = nodeName
 
-		// get the node configs right
-		nodeConfig := defaultNodeConfig
-		if opts.nodeConfigParam != "" {
-			nodeConfig = opts.nodeConfigParam
+		var mergedConfig map[string]interface{}
+		mergedConfig, err = mergeNodeConfig(defaultConfig, globalConfig, opts.customNodeConfigs[nodeNames[i]])
```

Comment:
A probably useful function can be created that merges any slice of maps with the expected precedence.
Custom node config shall be unmarshalled externally in that case.

Reply:
Already updated